src/config/ConfigService.js

```diff
@@ -18,23 +18,68 @@ export default class ConfigService extends Service {
 
 	initialize() {
 		// Initialization of dynamic configuration
-		const headerLogoFull = document.getElementsByName('header-logo-full')[0]?.content;
-		const headerLogoMini = document.getElementsByName('header-logo-minimized')[0]?.content;
+		const headerLogoFullLight = document.getElementsByName('header-logo-full-light')[0]?.content;
+		const headerLogoMiniLight = document.getElementsByName('header-logo-minimized-light')[0]?.content;
+		const headerLogoFullDark = document.getElementsByName('header-logo-full-dark')[0]?.content;
+		const headerLogoMiniDark = document.getElementsByName('header-logo-minimized-dark')[0]?.content;
 		const title = document.getElementsByName('title')[0]?.content;
 		const customCSS = document.getElementsByName('custom-css-file')[0]?.content;
 
 		let dynamicConfig = {};
-		let brandImage = {};
-		// Add custom header full logo
-		if ((headerLogoFull != undefined) && (headerLogoFull != "")) {
-			brandImage["full"] = headerLogoFull;
-			dynamicConfig["brand_image"] = brandImage;
+
+		// Add custom header full light logo
+		if ((headerLogoFullLight != undefined) && (headerLogoFullLight != "")) {
+			dynamicConfig.brandImage = {light: {full: headerLogoFullLight}};
+		}
+		// Add custom header minimized light logo
+		if ((headerLogoMiniLight != undefined) && (headerLogoMiniLight != "")) {
+			if (dynamicConfig?.brandImage?.light !== undefined) {
+				dynamicConfig.brandImage.light.minimized = headerLogoMiniLight;
+			} else {
+				dynamicConfig.brandImage = {light: {minimized: headerLogoMiniLight}}
+			}
+		}
+		// Add custom header full dark logo
+		if ((headerLogoFullDark != undefined) && (headerLogoFullDark != "")) {
+			if (dynamicConfig?.brandImage !== undefined) {
+				dynamicConfig.brandImage.dark = {full: headerLogoFullDark};
+			} else {
+				dynamicConfig.brandImage = {dark: {full: headerLogoFullDark}}
+			}
+		}
+		// Add custom header minimized dark logo
+		if ((headerLogoMiniDark != undefined) && (headerLogoMiniDark != "")) {
+			if (dynamicConfig?.brandImage?.dark !== undefined) {
```

Comment:
@petrKavulok Dont understand why you need to have the conditions and value assignment this way

cant you just do:

```
dynamicConfig["brandImage"] = {"dark": {"minimized": headerLogoMiniDark}};
```